hawkbit-ui/src/main/java/org/eclipse/hawkbit/ui/colorPicker/ColorPickerHelper.java

```diff
@@ -0,0 +1,74 @@
+/**
+ * Copyright (c) 2015 Bosch Software Innovations GmbH and others.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.eclipse.hawkbit.ui.colorPicker;
+
+import org.eclipse.hawkbit.ui.management.tag.SpColorPickerPreview;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.vaadin.shared.ui.colorpicker.Color;
+import com.vaadin.ui.Slider.ValueOutOfBoundsException;
+
+public class ColorPickerHelper {
+
+    private static final Logger LOG = LoggerFactory.getLogger(ColorPickerHelper.class);
+
+    /**
+     * Get color picked value in string.
+     *
+     * @return String of color picked value.
+     */
+    public static String getColorPickedString(final SpColorPickerPreview preview) {
+
+        return "rgb(" + preview.getColor().getRed() + "," + preview.getColor().getGreen() + ","
+                + preview.getColor().getBlue() + ")";
+    }
+
+    /**
+     * Covert RGB code to {@Color}.
+     *
+     * @param value
+     *            RGB vale
+     * @return Color
+     */
+    public static Color rgbToColorConverter(final String value) {
+
+        if (!value.startsWith("rgb")) {
```

Comment:
This may cause a `NullPointerException` later on. Either throw `IllegalArgumentException` or return `Optional`. I'd prefer an exception